aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModuleAgent.java

```diff
@@ -862,6 +863,9 @@ void onNewLeadershipTerm(
             throw new AgentTerminationException();
         }
 
+        final long nowNs = clusterClock.timeNanos();
+        timeOfLastLeaderMessageReceivedNs = nowNs;
```

Comment:
One question for this logic; how protective should we be against byzantine failures?  Should we check that the newLeadershipTerm/commitPosition message received is from the node that we currently consider to be the leader?

Reply:
It does not work, i.e. if we were to check for the `leadershipTermId` then we would not update the timeout and as a result there would be no delay applied in the election. Instead we trust any node that is currently a leader and update the time a message was received. Otherwise the case of an old valid leader with two empty follower nodes will not work.

Reply:
timeOfLastLeaderMessageRecievedNs -> timeOfLastLeaderUpdateNs

Reply:
Done.